### ISSUE TITLE
import_column_family bug

### ISSUE THREAD

**Issue by x-x-p** [facebook/rocksdb#6753]
when sorted_files.size() is 0,  sorted_files.size() - 1 is overflow， sorted_files[i + 1] operator[] fatal

https://github.com/facebook/rocksdb/blob/e04f3bce4fbe0453457047ea36e858888f0343e9/db/import_column_family_job.cc#L65-L70

**Comment by zhichao-cao** [comment]
@x-x-p Yes, you are right, size_t is unsigned. Will create a PR to fix it.
